package.json

```diff
@@ -46,11 +46,11 @@
     "jest-environment-node": "^25.1.0"
   },
   "optionalDependencies": {
-    "jest-dev-server": "*",
```

Comment:
Is this an optional dependency? feels like it's a `devDependency`.

Reply:
@mxschmitt I think it should be optional, cause we need it if only **server** option defined in config

Reply:
ah yep, I agree! :)